ovnimp.go

```diff
@@ -210,7 +210,12 @@ func (odbi *ovndb) populateCache(updates libovsdb.TableUpdates) {
 			odbi.float64_to_int(row.New)
 
 			if !reflect.DeepEqual(row.New, empty) {
+				if reflect.DeepEqual(row.New, odbi.cache[table][uuid]) {
```

Comment:
Do you need to check for `row, ok := odbi.cache[table][uuid]` before calling L213?

Reply:
I tried that DeepEqual handles it well, DeepEqual(<something not nil>, nil) returns false, so I think it doesn't require an extra check here. What do you think?